frontend/public/style/_forms.scss

```
@@ -10,3 +10,17 @@
 .co-form-subsection {
   margin-left: 20px;
 }
+
+.co-storage-class-dropdown-flex-column {
+  display: flex;
+  flex-direction: column;
+}
+
+.co-storage-class-dropdown-flex-row {
+  display: flex;
+}
+
+.co-storage-class-dropdown-icon-column {
+  flex: 30px;
```

I'd be explicit and use `flex-basis` if only setting one value.

```suggestion
  flex-basis: 30px;
```